templates/source/access-points.html

```diff
@@ -117,8 +117,17 @@
                 </div>
             </div>
             <div class="form-group">
-                <div class="col-sm-12">
+                <div class="col-sm-12 text-right">
                     <br />
+                    <button type="button" class="btn btn-danger cancel-button">
```

Comment:
If we use `btn` classes on an `a` element here, we can avoid having to add extra CSS to restyle the button to avoid default `a` styles:

```suggestion
                    <a href="{% url 'add-access-point' source.uuid %}" class="btn btn-danger">
```